src/mod.ts

```diff
@@ -1,5 +1,6 @@
 export const BUILDER_TAG = "deno_slack_builder@0.0.10";
 export const RUNTIME_TAG = "deno_slack_runtime@0.0.6";
+export const HOOKS_TAG = "deno_slack_hooks@0.0.1";
```

Comment:
Why is this `0.0.1`? It looks like the latest `deno_slack_hooks` is now `0.0.6`. Is it possible to use this packages tagged version instead of hardcoding the hooks version?

Reply:
Won't including this self-referencing tag guarantee that this will repo be broken before a release? In order to release a new version of the library, we'll need to bump this tag before the tag for this version exists on GitHub & deno.land

Reply:
Hmm, hard for me to picture the issue @shapirone, but I'm slow today 😛 . When releasing, we would update the version string, commit that update with a message like 'v1.0.0' or whatever version it is, tag that commit as v1.0.0, then push all those up to github. Once pushed, that will automatically deploy to deno.land. Now anyone can consume this latest version, and the self-referencing should still work?

Reply:
To wrap up the discussion around Neil's last point: the concern was final testing prior to a release. To address this, we are making clear what the releasing steps are in an attached maintainer's guide. The key is to do any testing _before_ bumping the version string we have been discussing.